tracker_package/tracker/src/tracker/tracker.py

```diff
@@ -0,0 +1,122 @@
+import rospy
+import select
+import cv2
+import numpy as np
+import socket
+import sys
+import pickle
+import struct
+import torch
+
+from ltr.data.bounding_box_utils import masks_to_bboxes
+from pytracking.evaluation import Tracker
+from sensor_msgs.msg import Image
+from std_msgs.msg import Float32MultiArray
+from cv_bridge import CvBridge, CvBridgeError
+from tracker.optical_flow import VisualTrackerKLT   
+
+
+class ObjectTracker:
+
+    def __init__(self):
+        self.clientsocket = socket.socket(socket.AF_INET,socket.SOCK_STREAM)
+        try:
+            self.clientsocket.connect((rospy.get_param('tracker/ip','172.17.0.1'), int(rospy.get_param("tracker/port",5556))))
```

Comment:
The usage of these params is done several times throughout your class. Store these in variables instead of getting them multiple times from the params server.